spec/controllers/annotations_controller_spec.rb

```diff
@@ -273,6 +273,29 @@
         assert_response :success
         expect(anno.annotation_text.reload.content).to eq 'new content'
       end
+
+      it 'successfully updates a singular annotation text' do
+        anno1 = create(
+          :text_annotation,
+          submission_file: submission_file,
+          creator: user,
+          result: result
+        )
+        anno2 = create(
```

Comment:
I don't think this does what you want: the `:text_annotation` factory automatically creates a new `annotation_text` instance. So here you want to pass in an option for `annotation_text` to make it the same as the one for `anno1`.